src/app/(BottomTabNavigation)/Cases/QRCodeScanner/AddCase/index.tsx

```diff
@@ -0,0 +1,59 @@
+import { useLocalSearchParams, useRouter } from 'expo-router';
+import { useEffect, useState } from 'react';
+import { Image, Text, TouchableOpacity, View } from 'react-native';
+
+import styles from './styles';
+import {
+  containsDuplicateCase,
+  getImageUrl,
+  uploadCase,
+} from '../../../../../supabase/queries/cases';
+import { CaseUid } from '../../../../../types/types';
+
+function AddCase() {
+  const { id, title, summary } = useLocalSearchParams() as {
+    id: CaseUid;
+    title: string;
+    summary: string;
+  };
+  const router = useRouter();
+  const [loading, setLoading] = useState<boolean>(true);
+  const [image, setImage] = useState<string>();
+  console.log(loading);
+
+  useEffect(() => {
+    const waitForImage = async () => {
+      const bucketImage = await getImageUrl(id);
+      setImage(bucketImage);
+      setLoading(false);
+    };
+    waitForImage();
+  }, []);
+
+  const addToCases = async () => {
+    uploadCase(id);
+    router.push('/Cases');
+  };
+  if (loading) {
+    return (
+      <View style={styles.loading}>
```

Comment:
I personally don't see the loading screen while waiting for the image. It directly loads the summary while it waits for the image to load.

Reply:
me neither :( can we debug this tmrw?

Reply:
also writing this here for now: how do I get the impactfund banner to show up instead of my ugly ass route on the header?

Reply:
We need to create a `_layout.tsx` file for the `QRCodeScanner` route. Refer to `src/app/(BottomTabNavigation)/Cases/_layout.tsx`